addons/binding/org.openhab.binding.tivo/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,152 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<thing:thing-descriptions bindingId="tivo"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 
+           http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <thing-type id="sckt">
+        <label>TiVo DVR Controller</label>
+        <description>Monitor and control your TiVo via DIRECT SOCKET commands leveraging the TIVO protocol 1.1 specification.  The TiVo TCP Control Protocol is an ASCII-based command protocol for remote control of a TiVo DVR over a TCP local network connection (the WAN or cable network interface cannot be used). The commands allow
+control of channel changes, user interface navigation and allow the client to send simulated remote control button presses to the Digital Video Recorder.<![CDATA[ <br /><br /> ]]>See https://www.tivo.com/assets/images/abouttivo/resources/downloads/brochures/TiVo_TCP_Network_Remote_Control_Protocol.pdf for supported commands / codes for each channel.</description>
+
+        <channels>
+            <channel id="tivoChannelForce" typeId="tivoChannelForce"/>
+            <channel id="tivoChannelSet" typeId="tivoChannelSet"/>
+            <channel id="tivoTeleport" typeId="tivoTeleport"/>
+            <channel id="tivoIRCommand" typeId="tivoIRCommand"/>
+            <channel id="tivoKBDCommand" typeId="tivoKBDCommand"/>
+            <channel id="tivoStatus" typeId="tivoStatus"/>
+            <channel id="tivoCommand" typeId="tivoCommand"/>
+        </channels>
+        
+        <config-description>
+        	<parameter name="deviceName" type="text" required="true">
+				<default>My Tivo</default>
+				<required>true</required>
+				<label>Device Name</label>
+				<description>A friendly name to refer to this device.<![CDATA[ <b> ]]>Default: Device name specified on the TiVo.<![CDATA[ </b> ]]></description>
```

Comment:
Both are true!  When created by discovery the box name is used.  When added manually or by .thing file the default My Tivo is used (unless that name parameter is explicitly specified etc.).  I'll tweak the text.